requirements.txt

```diff
@@ -0,0 +1,31 @@
+appdirs==1.4.0
```

Comment:
That's as slim as it gets. That's the output of `pip freeze` having installed only gmusicapi and future.

> On Jul 27, 2016, at 8:28 PM, John Elkins <notifications@github.com> wrote:
> 
> In requirements.txt:
> 
> > @@ -0,0 +1,31 @@
> > +appdirs==1.4.0
> can this be slimmed down so that it has what the playlist scripts directly depends on?
> 
> looks like for now it may just be gmusicapi and future.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>